packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.ts

```diff
@@ -767,3 +768,31 @@ export function getIAMPolicies(resourceName: $TSAny, crudOptions: $TSAny) {
 
   return { policy, attributes };
 }
+
+export const openConsole = async (context?: $TSContext) => {
+  const amplifyMeta = stateManager.getMeta();
+  const bucketName = getS3BucketNameFromAmplifyMeta(amplifyMeta);
+
+  if (bucketName) {
+    let url = `https://s3.console.aws.amazon.com/s3/buckets/${bucketName}`;
+    open(url, { wait: false });
+  } else {
+    printer.error('There are no Amazon S3 storage resources pushed to the cloud');
+  }
+};
+
+function getS3BucketNameFromAmplifyMeta(amplifyMeta: $TSMeta): string | null {
```

Comment:
We could do that, but just double-checking, because the dependant function getS3ResourcesFromAmplifyMeta(..) and its dependencies are also in this same file, so it looked tidier for me to keep them here. What do you think?

See https://github.com/aws-amplify/amplify-cli/blob/01ff80f5630ee0389d5ba06344ebb2d70be926d2/packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.ts#L421 and https://github.com/aws-amplify/amplify-cli/blob/01ff80f5630ee0389d5ba06344ebb2d70be926d2/packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.ts#L403

Should we move those as well? Or keep them where they are?

Reply:
I didn't realize there were other similar helper functions in this file, so we can leave it as is then. I think they should all be moved at some point, but that's out of scope for this PR.